### PR TITLE
Add credits volume client preference. Credits are 25% quieter by default.

### PR DESCRIPTION
**Pull request by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE**
:cl:
 * rscadd: Added a 'credits volume' client preference. Credits are 25% quieter by default.

### CODE REVIEW

File: code/modules/credits/episode_name.dm [hunk 1]
```diff
@@ -376,6 +376,12 @@
 			if(piggycount > 3)
 				episode_names += new /datum/episode_name/rare("BRINGING HOME THE BACON", "[piggycount] little piggies went to the shuttle...", min(1500, piggycount*200))
 
+			var/cowcount = 0
+			for(var/mob/living/simple_animal/cow/C in shuttle)
+				cowcount += 1
+			if(cowcount > 1)
+				episode_names += new /datum/episode_name/rare("WHEN THE COWS COME HOME", "There were [cowcount] cows on the shuttle.", min(1500, cowcount*300))
```
Comment (MadmanMartian) [inline code/modules/credits/episode_name.dm]: jej

**Comment by cowboybuttsdrivemenuts** [comment]
based

File: code/modules/credits/episode_name.dm (hunk 1 above)
Comment (SonixApache) [reply code/modules/credits/episode_name.dm]: tis unatomic, but

**Comment by SonixApache** [comment]
thank god

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
@MadmanMartian updated

**Pull request merged**